src/app/actions/permissions.ts

```diff
@@ -121,6 +121,9 @@ export async function removePermission({
  * @returns List of all permissions
  */
 export async function getPermissions(): Promise<Permission[]> {
+  // wait 1 second
+  await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Why are you waiting 1 second here? If you want to delay the execution of this query you can do this in the `useQuery` hook.